oak/server/dev/dev_oak_runner_main.cc

```diff
@@ -36,21 +36,20 @@ int main(int argc, char* argv[]) {
   std::unique_ptr<oak::ApplicationConfiguration> application_config =
       oak::ReadConfigFromFile(absl::GetFlag(FLAGS_application));
 
-  asylo::StatusOr<oak::CreateApplicationResponse> result =
-      loader->CreateApplication(*application_config);
-  if (!result.ok()) {
+  grpc::Status status = loader->CreateApplication(*application_config);
+  if (!status.ok()) {
     LOG(ERROR) << "Failed to create application";
     return 1;
   }
-  oak::CreateApplicationResponse response = result.ValueOrDie();
   std::stringstream address;
-  address << "0.0.0.0:" << response.grpc_port();
-  std::string application_id(response.application_id());
-  LOG(INFO) << "Oak Application id=" << application_id << ": " << address.str();
+  address << "0.0.0.0:" << application_config->grpc_port();
```

Comment:
Should we check for the port number to be >0 here?

Reply:
It's already checked:
https://github.com/project-oak/oak/blob/7978a6eda87df6be569505c2bab9b4f1d0a0aa1e/oak/server/oak_runtime.cc#L65-L70

Reply:
Is that check correct though? AFAICT it will allow ports like -1, 0, and 1025, is that the intention?

Reply:
Will change it to `grpc_port <= 1023`